policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml

```diff
@@ -0,0 +1,12 @@
+description: California CalWORKs counts these property sources as resources.
+
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs TANF resources
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
+    - property_value
```

Comment:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/1522c989-0bb4-4611-bed3-7155b45b02a7)

I missed this:
The personal property includes other items such as cash etc. we need to see how much of that we can include given inputs that we have